pkg/render/render_test.go

```diff
@@ -0,0 +1,114 @@
+package render
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("RenderTemplate", func() {
+	Context("when a valid YAML manifest is given as a parameter", func() {
+		It("should successfully render", func() {
+			renderData := MakeRenderData()
+
+			renderedFromYAML, err := RenderTemplate("testdata/simple.yaml", &renderData)
+			Expect(err).NotTo(HaveOccurred())
+
+			Expect(renderedFromYAML).To(HaveLen(1))
+			expected := `
```

Comment:
As I understand this is testing RenderTemplate that converts yaml manifest to the structs from the api, so it's better to write the "expected"·data as structs too.

Reply:
Done